jtuples/src/test/java/org/jtuples/PairTest.java

```diff
@@ -135,6 +135,30 @@ public void testShiftLeftReturnsNew() {
     }
 
     @Test
+    public void testShiftLeftWithValue() {
+        expected = new Pair<>("2", "3");
+
+        assertEquals(expected, pair.shiftLeft("3"));
+    }
+
+    @Test
+    public void testShiftLeftWithValueReturnsNew() {
+        assertNotEquals(pair, pair.shiftLeft("3"));
+    }
+
+    @Test
+    public void testShiftLeftWithNull() {
+        expected = new Pair<>("2", null);
+
+        assertEquals(expected, pair.shiftLeft(null));
+    }
+
+    @Test
+    public void testShiftLeftWithNullReturnsNew() {
```

Comment:
@git-afsantos Is this necessary? It's already tested in `testShiftLeftWithValueReturnsNew` and `testShiftLeftWithNull`.